btf/handle.go

```diff
@@ -110,7 +110,10 @@ func NewHandleFromID(id ID) (*Handle, error) {
 }
 
 // Spec parses the kernel BTF into Go types.
-func (h *Handle) Spec() (*Spec, error) {
+//
+// base must contain type information for vmlinux if the handle is for
+// a kernel module. It may be nil otherwise.
+func (h *Handle) Spec(base *Spec) (*Spec, error) {
```

Comment:
Do we have a policy regarding breaking an API like this?

Reply:
Do we make a note somewhere to document the change in the release notes? Yes, adding it to CONTRIBUTING is a good idea, maybe even add that breakages should always be pointed out/argued in commit messages?

Reply:
See #1003

Reply:
> Breaking the API in a way that causes compile failure is OK

Caveat: ... if an equivalent API is available and the migration steps are clear, e.g. in the case of deprecations.